docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 |                     |                                                                                                                                                                                                                       | year -- vehicle year production                                                                                                                                                                                     |
 |                     |                                                                                                                                                                                                                       | model -- car model                                                                                                                                                                                                  |
 |                     |                                                                                                                                                                                                                       | color -- car color                                                                                                                                                                                                  |
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| CarSellOrderInfo    | Information about car’s selling as well as proof of a current car owner. Used in transaction processing.                                                                                                              | carBoxToOpen -- car box for start selling                                                                                                                                                                           |
+| CarSellOrderInfo    | Information about the car being sold as well as proof of ownership of seller. Used in transaction processing.                                                                                                         | carBoxToOpen -- car box for start selling                                                                                                                                                                           |
```

Comment:
@zennermac 

| CarSellOrderInfo    | Is information about the car being sold as well as proof of ownership of seller. Used in transaction processing.                                                                                                        
 | carBoxToOpen -- A CarBox to initiate a sale                                                                                                                                                                         |